src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
So I basically relied on createIndexes method that already existed in the JdbcAccessMethod class. These methods did not have the ability to specify index order. Should I add this feature to the createIndex methods and let users specify sorting order through the createIndex encoding? I don't think this would be difficult to do....Although I'm not yet sure how users should specify this order